packages/jest-runtime/src/__tests__/test_root/test_json_preprocessor.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+module.exports.process = () => `{"foo": "foo"}`;
```

Comment:
Does this mean that the preprocessor is expected to return a JSON string instead of an `Object`?

Is it also GIVEN a JSON string to process?

Can you make sure the test covers actually transforming something (maybe make a value uppercase on the `foo` key)?

Reply:
Yes, it's given the source as a string, and returns a transformed source string, just as for other transforms.

I've changed the test preprocessor to actually process the string to make it clearer.